dexter/src/main/java/com/karumi/dexter/Dexter.java

```diff
@@ -52,15 +56,25 @@ public static void checkPermission(String permission, PermissionListener listene
     instance.checkPermission(permission, listener);
   }
 
-  static void onActivityCreated(Activity activity) {
-    instance.onActivityCreated(activity);
+  /**
+   * Checks the permissions and notifies the listener of its state
+   * It is important to note that permissions still have to be declared in the manifest
+   *
+   * @param permissions Collection of values found in {@link android.Manifest.permission}
+   * @param listener The class that will be reported when the state of the permissions are ready
+   */
+  public static void checkPermissions(Collection<String> permissions,
```

Comment:
We should also change the input type to a java ``Set`` before to pass this information to dexter to avoid problems with duplicated permissions inside the collection.